carbon-aware-sdk/license.md

```diff
@@ -1,39 +0,0 @@
-Materials in this repository other than source code are provided as follows:
```

Comment:
@Willmish Did you mean to delete this license?  Either way it is linked in the [readme.md](https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/dev/README.md?plain=1#L43) file which should be updated.

Reply:
Hi @bderusha, I have seen your PR at https://github.com/Green-Software-Foundation/carbon-aware-sdk/pull/105. 
We should prabably move the license following the guidance from #105.
Would we move it here or in #105  please? If you are happy, let's get that merge and update 105 to have the license in the place.
(Will that also help feed into this documentation better? https://github.com/Green-Software-Foundation/carbon-aware-sdk/issues/41 - I am not I understand where the guidance is on how to ensure that online documentation gets updated inline with the repo)